homeassistant/components/media_player/webostv.py

```diff
@@ -100,6 +112,7 @@ def request_configuration(host, hass, add_devices):
     # pylint: disable=unused-argument
     def lgtv_configuration_callback(data):
         """The actions to do when our configuration callback is called."""
+        # pylint: disable=no-value-for-parameter
```

Comment:
Don't add this. You are actually introducing a bug. You added a parameter to `setup_tv` and you are not passing that in here.